packages/flutter_tools/bin/macos_assemble.sh

```diff
@@ -93,3 +93,12 @@ RunCommand "${FLUTTER_ROOT}/bin/flutter"                                    \
     --build-outputs="${build_outputs_path}"                                 \
     --output="${ephemeral_dir}"                                             \
    "${build_mode}_macos_bundle_flutter_assets"
+
+if [[ $? -ne 0 ]]; then
```

Comment:
Hopefully no one ever adds `set -e` to the top like https://github.com/flutter/flutter/pull/50664.